File: rb/utility/make_context.rb
# KokkaiKaigirokuApi SDK utility: make_context
require_relative '../core/context'
module KokkaiKaigirokuApiUtilities
  MakeContext = ->(ctxmap, basectx) {
    KokkaiKaigirokuApiContext.new(ctxmap, basectx)
  }
end

File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// KokkaiKaigirokuApi SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class KokkaiKaigirokuApiMakeContext
{
    public static function call(array $ctxmap, ?KokkaiKaigirokuApiContext $basectx): KokkaiKaigirokuApiContext
    {
        return new KokkaiKaigirokuApiContext($ctxmap, $basectx);
    }
}
